config/v1/types_infrastructure.go

```diff
@@ -536,6 +551,18 @@ type AWSPlatformStatus struct {
 	// +optional
 	// +nullable
 	CloudLoadBalancerConfig *CloudLoadBalancerConfig `json:"cloudLoadBalancerConfig,omitempty"`
+
+	// ipFamily specifies the IP protocol family that should be used for AWS
+	// network resources. This controls whether AWS resources are created with
+	// IPv4-only, or dual-stack networking with IPv4 or IPv6 as the primary
+	// protocol family.
+	//
+	// +default="IPv4"
```

Comment:
Remind me why we default this to IPv4? VS leaving it empty on upgrade?

Reply:
The underlying assumption is that all existing running clusters are are IPv4 based.